src/service/inversionSolution.service.ts

```diff
@@ -40,5 +42,34 @@ export const filterData = (
       xy.push(object);
     }
   });
+
   return minDataFilter(xy);
 };
+
+export const getHazardTableOptions = (data: InversionSolutionHazardTabQueryResponse): HazardTableOptions => {
+  const rows = data?.node?.rows;
+
+  const forecastTimes = new Set<string>();
+  const bgSeismicity = new Set<string>();
+  const pga = new Set<string>();
+  const gmpe = new Set<string>();
+  const locations = new Set<string>();
+
+  rows?.map((row) => {
+    if (row && row !== null) {
```

Comment:
if (row) already tests for nullability so row !== null is a bit redundant

Reply:
Thanks that's true changed it now